adaptive_components/lib/src/adaptive_container.dart

```diff
@@ -61,7 +63,19 @@ class AdaptiveContainer extends StatelessWidget {
   ///
   /// This is used by the builder to see what type of screen the user wants this
   /// [AdaptiveContainer] to fit within.
-  final AdaptiveConstraints adaptiveConstraints;
+  AdaptiveConstraints adaptiveConstraints;
+
+  /// AdaptiveColumn is used with [AdaptiveColumn] and [AdaptiveRow] to represent
```

Comment:
I don't think there is a class called `AdaptiveRow`?

Reply:
Done